src/pretix/control/forms/global_settings.py

```diff
@@ -91,3 +94,4 @@ class UpdateSettingsForm(SettingsForm):
     def __init__(self, *args, **kwargs):
         self.obj = GlobalSettingsObject()
         super().__init__(*args, obj=self.obj, **kwargs)
+
```

Comment:
Avoid adding spurious empty lines, since it makes syncing with other patches more difficult.
So please revert this.